staging/src/k8s.io/component-base/config/leaderelectionconfig.go

```diff
@@ -14,15 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package leaderelectionconfig
+/*
+NOTE: this file has been copied from k8s.io/kubernetes/pkg/client/leaderelectionconfig.
+*/
+package config
```

Comment:
Didn't realize earlier that this package contains API types, this probably doesn't belong here then? Wondering if we should create a new package `k8s.io/component-base/leaderelection` or a subpackage in config like `k8s.io/component-base/config/options` or `k8s.io/component-base/config/flags`?

Any opinions @liggitt @mtaufen?

Reply:
`k8s.io/component-base/config/options` makes sense to me since the LeaderElectionConfiguration type lives in `k8s.io/component-base/config`

Reply:
Moved to k8s.io/component-base/config/options